### DISCUSSION TITLE
Yarn Berry support?

### DISCUSSION THREAD

**Discussion by yovanoc** [prisma/prisma#4127] [Q&A] [answered]
Hello,

I tried in a nextjs project with yarn berry as package manager to use @nexus/schema with nexus-plugin-prisma (all lasts versions) and even if I have installed @prisma/client and @prisma/cli I got this error and I don't know where to report it. Thanks

<img width="818" alt="image" src="https://user-images.githubusercontent.com/7302317/97889397-a2153f00-1d2c-11eb-9023-5dfd9dbb2699.png">

**Comment by ryands17** [comment, marked as answer]
Yarn 2 is still yet to be supported. You can find and follow the related request here: https://github.com/prisma/prisma/issues/1439

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
